extra/release_tool.py

```diff
@@ -2586,6 +2586,92 @@ def determine_version_to_include_in_release(state, repo):
     return True
 
 
+def run_and_combine_statistics_and_changelog(
+    workdir, single_repo, from_version, to_version, output_file
+):
+    """Generate statistics and changelogs and combine them in a single file"""
+
+    release_date = execute_git(
+        None,
+        workdir if single_repo else integration_dir(),
+        [
+            "for-each-ref",
+            r"--format=%(taggerdate:format:%m.%d.%Y)",
+            "refs/tags/" + to_version,
+        ],
+        capture=True,
+    )
+    release_name = "%s %s" % (
+        "Mender" if not single_repo else os.path.basename(workdir),
+        to_version,
+    )
+    release_header = "## %s\n\n_Released %s_\n\n" % (release_name, release_date)
+
+    command_args = []
+    if single_repo:
+        command_args.append("--repo")
+    else:
+        command_args.append("--base-dir")
+        command_args.append(workdir)
+    if from_version:
+        command_args.append("%s..%s" % (from_version, to_version))
+    else:
+        command_args.append(to_version)
+
+    changelog_tool = os.path.join(
+        integration_dir(), "extra/changelog-generator/changelog-generator"
+    )
+    statistics_tool = os.path.join(integration_dir(), "extra/statistics-generator")
+
+    changelog_out = subprocess.check_output(
+        [changelog_tool] + command_args, cwd=workdir
+    )
+    statistics_out = subprocess.check_output(
+        [statistics_tool] + command_args, cwd=workdir
+    )
```

Comment:
Can you also print the commands? This will help in the cases where the changelog-generator complains about ambiguous entries.

Reply:
Done. See comment below too.